pnpbridge/src/adapters/src/modbus_pnp/ModbusPnp.h

```diff
@@ -92,12 +92,14 @@ typedef int SOCKET;
         HANDLE hDevice;
         LOCK_HANDLE hConnectionLock;
         IOTHUB_DEVICE_CLIENT_HANDLE DeviceClient;
```

Comment:
>DeviceClient [](start = 36, length = 12)

Since the adapters are re-compiled for edge module support already, another option to consider is define a custom type for the connection handle along with macros for IoTHub<Device|Module>Client_Foo. That way the adapter code will largely remain unchanged.